Gopkg.lock

```diff
@@ -66,6 +66,12 @@
   revision = "47dc60e71eed504e3ef8e77ee3c6fe720f3be57f"
   version = "v1.3.0"
 
+[[projects]]
+  name = "github.com/ghodss/yaml"
```

Comment:
Ah just noticed. Curious why this yaml package specifically?

Reply:
ahh I'm seeing the doc. very cool!

Reply:
I was a bit skeptical about this library, but then I saw that kubernetes is using it: https://github.com/kubernetes/kubernetes/issues/2600